cfgov/unprocessed/apps/prepaid-agreements/css/search-bar.less

```diff
@@ -0,0 +1,125 @@
+// The code is borrowed, with some modifications, from 
+// ccdb5-ui's App.less, SearchBar.less, and select.less
+
+.layout-row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.layout-column {
+    display: flex;
+    flex-direction: column;
+}
+
+.flex-fixed {
+    flex: none;
+}
+
+.flex-all {
+    flex: 1 0 auto;
+}
+
+button.a-btn.flex-fixed {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+}
+
+.lt-ie10 {
+    button.a-btn.flex-fixed {
+        border-radius: .25em;
+    }
+
+    .a-select {
+        margin-bottom: unit( @grid_gutter-width * 2/3 / @base-font-size-px, em );;
+    }
+
+    .flex-all {
+        margin-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );;
+    }
+    .respond-to-min( @bp-med-min, {
+        .a-select,
+        .flex-all {
+            margin-bottom: 0;
+        }
+        .layout-row {
+            display: table;
+            width: 100%;
+
+            .a-select{
+                display: table-cell;
+                width: 25%;
+            }
+
+            .flex-all {
+                width: 100%;
+                display: table-cell;
+            }
+
+            .a-btn {
+                display:table-cell;
+                border-top-left-radius: 0;
+                border-bottom-left-radius: 0;
+            }
+        }
+    });
+}
+
+.respond-to-max( @bp-sm-max, {
+    .a-select {
+        margin-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+        width: 100%;        
+    }
+});
+
+ // select
+@select-height:     @size-ii;
```

Comment:
TRIVIAL: Space after colon not needed.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96